website/static/css/file-view-editor.css

```diff
@@ -0,0 +1,25 @@
+/* Style for Ace wiki editor */
+.wiki-editor {
```

Comment:
Would it make sense to add these rules to `file-view-page.css`? That would save an extra request for a stylesheet.